zero/gkyl_gk_geometry.h

```diff
@@ -39,6 +39,7 @@ struct gk_geometry {
   struct gkyl_array* gyyj;
   struct gkyl_array* gxzj;
   struct gkyl_array* eps2; // eps2 = Jg^33 - J/g_33
+  struct gkyl_array* bmag_mid;
```

Comment:
@akashukla Can you add comments explaining all the entries in this struct please?